.github/workflows/build-geo.yml

```diff
@@ -0,0 +1,37 @@
+name: build-geo
+on:
+  workflow_call:
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.workflow_run.head_branch }}
+  cancel-in-progress: true
+jobs:
+  docker:
+    name: Build and push Docker image
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_NAME: ghcr.io/eukarya-inc/reearth-plateauview/plateauview-geo:latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Build and load docker image
+        uses: docker/build-push-action@v3
+        with:
+          context: ./geo
+          platforms: linux/amd64
+          load: true
+          tags: ${{ env.IMAGE_NAME }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+          push: false
+      - name: Save docker image
```

Comment:
@keiya01 Don't we want to maintain versions?
if we do, it might be a good idea to add something like [this](https://github.com/reearth/reearth/blob/2ebdf5f1b05b4953ddf25e6dc45a8bac15d01148/.github/workflows/build_web.yml#L109).

Reply:
I guess this project will be the maintenance mode after this project is completed like VIEW2.0. So I think we don't need to manage version for now:+1: